starter-code/json-hw.js

```diff
@@ -19,14 +18,21 @@ with the date the photo was taken. The first console.log should return the follo
 
 */
 
+flickerObj.element.forEach(function(elem){
```

Comment:
The name of an array you will iterate over it is `flickerObj.items` not `flickerObj.element`